float64 future_time
float64 weight
float64 radius
geometry_msgs/Point obstacle_point

# row-major 3x3 covariance matrices
float64[9] obstacle_covariance
